store/src/java/com/zimbra/cs/index/DBQueryOperation.java

```diff
@@ -787,10 +791,26 @@ private void luceneFirstGetNextChunk(SortBy sort) throws ServiceException {
             } else {
                 // must not ask for offset,limit here b/c of indexId constraints!,
                 dbSearch(dbHits, sort, -1, -1);
+                //if we are sorting by relevance, we have to re-sort the DB results here
+//                if (sort.getKey() == SortBy.Key.RELEVANCE) {
+//                    sortDbHitsByLuceneOrder();
+//                }
             }
```

Comment:
Would prefer not to see commented out code checked in.

Reply:
Oh, good catch. I meant to delete that.